### ISSUE TITLE
ES2024 resizable ArrayBuffer

### ISSUE THREAD

**Issue by p-bakker** [mozilla/rhino#1397]
Proposal: https://github.com/tc39/proposal-resizablearraybuffer

**Comment by camnwalter** [comment]
I was looking into this, and the changes themselves aren't difficult to add, it's just hard to run the tests. It's unfortunate because most of the 262 tests related to resizable array buffers use const inside for of loops as well as inside blocks, which we don't handle correctly.

**Comment by p-bakker** [comment]
Create a PR on the test262 testsuite repo to replace the const for let's 

They in general are open to such PRs

**Comment by rbri** [comment]
@p-bakker is your pr merged?

**Comment by p-bakker** [comment]
I was just suggesting to create a PR, I haven't created one

**Comment by rbri** [comment]
Hi @camnwalter,
still interested in this. Maybe we can work together here?

**Comment by camnwalter** [comment]
Yeah sure :) Haven't looked too much into it since I opened up this issue, but it looks pretty straightforward to do. It's just a bit tricky because I don't want to make the changes in 262 without testing with rhino making sure it actually helps, and adding resizability was also a bit more complex than I thought. Could just be me misremembering though...
